docs/contributors_agreement_explained.rst

```diff
@@ -73,7 +73,7 @@ so that I can contribute the same code to other projects under different terms o
 Does the Foundation control use of the Plone trademark?
 
   Yes.
-  In order to keep the trademark,
+  to keep the trademark,
```

Comment:
Since that is the start of a new sentence, "To" should be capitalized

Reply:
Right ! Would be easier to spot if people start to write for humans and ditch 'weird' coding styles for docs.. just saying ...

Reply:
or in his case here, follow and learn reST ...

Reply:
Yeah these Q & A formats aren't working on this page.